generators/bdk/python/templates/requirements.txt.ejs

```diff
@@ -1 +1,2 @@
 sym-api-client-python>=<%= bdkVersion %>
+Jinja2
```

Comment:
don't we have to set a version here?

Reply:
No version means pip installs the latest version.